instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/constants.py

```diff
@@ -0,0 +1,28 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# defaults
+APP = "redis"
+DEFAULT_SERVICE = "redis"
+
+# net extension
+DB = "out.redis_db"
+
+# standard tags
+RAWCMD = "redis.raw_command"
+CMD = "redis.command"
```

Comment:
These are only used in `patch.py`, better to move them there and make them private if they are defined as module-level constants.

Reply:
fixed